DbContext.js

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect, createContext } from "react";
+import { Text, View } from "react-native";
+import * as SQLite from "expo-sqlite";
+
+export const DBContext = createContext(undefined);
+
+export default function DataBaseContext({ children }) {
```

Comment:
This should be named `Provider` instead of Context, so `DBProvider` would be better.